routes/attachments.js

```diff
@@ -4,21 +4,19 @@
 const asyncMiddleware = require('../middleware/asyncMiddleware')
 const express = require('express')
 const router = express.Router()
-const AttachmentCoordinates = require('../lib/attachmentCoordinates')
 
 // Get a proposed patch for a specific revision of a component
 router.get('/:id', asyncMiddleware(getAttachment))
 
 async function getAttachment(request, response) {
-  const coordinates = new AttachmentCoordinates(request.params.id)
-  const result = await harvestStore.getAttachment(coordinates)
+  const result = await attachmentStore.get(request.params.id)
```

Comment:
either here or in the attachmentStore, we need to add the segment `attachment/` to the blob name.
This was being done in the `AttachmentCoordinates` before.

This `id` is just the hash and then the `azblobAttachmentStore` just does `name = id + '.json'`

Reply:
this is now in a separate container 👍